core/statistics/machine/netStatistics.go

```diff
@@ -112,3 +123,13 @@ func (ns *NetStatistics) PercentSent() uint64 {
 func (ns *NetStatistics) PercentRecv() uint64 {
 	return atomic.LoadUint64(&ns.percentRecv)
 }
+
+func getCurrentProcess() (*process.Process, error) {
```

Comment:
this should be in a separate file

Reply:
Done. Created a new file with name common.go and added in this file function getCurrentProcess().